firmware/Src/synth/Timbre.h

```diff
@@ -228,6 +228,10 @@ class Timbre {
     float low6 = 0, band6 = 0;
     float low7 = 0, band7 = 0;
     float low8 = 0, band8 = 0;
+    float low9 = 0, band9 = 0;
```

Comment:
Is it possible to use the same variables ? 
We'll soon be out of memory for the fast access RAM.

Reply:
yes, i'll do some refactoring tonight to clean up varaibles allocation.